clang/lib/CodeGen/CGBuiltin.cpp

```diff
@@ -2979,8 +2979,8 @@ RValue CodeGenFunction::EmitBuiltinExpr(const GlobalDecl GD, unsigned BuiltinID,
     case Builtin::BI__builtin_atan2l:
     case Builtin::BI__builtin_atan2f128:
       return RValue::get(emitBinaryMaybeConstrainedFPBuiltin(
-          *this, E, Intrinsic::atan2,
-          Intrinsic::experimental_constrained_atan2));
+          *this, E, Intrinsic::atan2, Intrinsic::experimental_constrained_atan2,
```

Comment:
@zahiraam , could you please take a look?

Reply:
@Fznamznon I think I had already reviewed this and approved it.

Reply:
thanks!

Reply:
This is cherry-pick of @zahiraam 's fix. :)